smart-contract/README.md

```diff
@@ -630,6 +630,61 @@ NOTE: you can get the address for `bank2` with the following command:
 provenanced keys show -a bank2 --home build/node0 -t
 ```
 
+You can now see that `bank2` holds $50 of `usdf.local`.
+
+```bash
+provenanced q bank balances tp145r6nt64rw2rr58r80chp70ejdyqenszpg4d47 -t -o json | jq
+{
+  "balances": [
+    {
+      "denom": "nhash",
+      "amount": "98000000000"
+    },
+    {
+      "denom": "usdf.local",
+      "amount": "5000"
+    }
+  ],
+  "pagination": {
+    "next_key": null,
+    "total": "0"
+  }
+}
+```
+
+However, `bank2` does not have sufficient reserve token to redeem `usdf.local` for `bank2.coin`.
+`bank2` must mint to cover before redemption.
+
+TODO: provide instructions on bank-to-bank settlement or holding `usdf.local` backed by `bank2.coin`.
+
+```bash
+provenanced tx wasm execute \
+    tp14hj2tavq8fpesdwxxcu44rty3hh90vhuz3ljwv \
+    '{"mint":{"amount":"5000","address":"tp145r6nt64rw2rr58r80chp70ejdyqenszpg4d47"}}' \
```

Comment:
notice the address specified here is the bank2 address so bank2 will hold usdf.local that needs to be converted to fiat at bank1 via bank settlement

Reply:
phil is working on reporting for all usdf movement but ultimately you should be able to query pretty easily who has reserve token at markers to redeem against within the system